### DISCUSSION TITLE
BlueJay U1 coming soon!

### DISCUSSION THREAD

**Discussion by Navid200** [NightscoutFoundation/xDrip#3465] [Show and tell]
**Good news!**  
Soon, a new model of BlueJay watch will be released. 
https://bluejay.website/shop/product/bluejay-u1-27
![image](https://github.com/NightscoutFoundation/xDrip/assets/51497406/cc8c9fcb-2364-402a-a2d3-1c81587a000a)

This is currently in beta testing.  But, will be available to everyone soon.

It has major upgrades compared to GTS in my opinion:

It accepts a sim card.  So, it can upload to the cloud without a phone using WiFi or cell service.  USA 4G band is not available yet.    

It is a full Android watch.  So, if you want, you can use it for sending emails and text and effectively using it like a phone.  Of course, the small screen makes it difficult for text editing and battery use may go up too.  But, it is possible if needed.

It runs a full-fledged xDrip.    So, it also collects from G7 or Dexcom One.

**Comment by belfastraven** [comment]
My bluejay u1 arrived today.  I do have it reading a g7 and will set up nightscout uploads once I hook it up to a bluetooth keyboard.  Because of having to use a dexcom receiver, I have to turn off collection on phone if I use the watch..  Do you know of a good way to enable phone xdrip  to get data from watch.  It seems to work differently that the older bluejays in that respect

**Reply by Navid200** [reply]
I can provide some answers.  But, this thread was not created to become a jumble of questions and answers.
Please start a new thread.  Ask your question.  And those who can will provide comments.  Then, anyone searching online can find your thread or any other thread and see content that is directly related to the title.

**Comment by tynbendad** [comment]


**Reply by Navid200** [reply]
No, I don't know when the beta tests will be completed.
